src/camvidlog2/ha_mqtt_client.py

```
@@ -0,0 +1,113 @@
+        userdata: Any,
+        flags: dict[str, Any],
+        rc: mqtt.ReasonCode,
+        properties: mqtt.Properties | None,
+    ) -> None:
+        """Callback when the internal client has connected"""
+        if self._connect_event is None:
+            raise ValueError("Client not awaiting connection")
+        self._connect_event.set()
+
+    def _on_message(
+        self, client: mqtt.Client, userdata: Any, msg: mqtt.MQTTMessage
```

Ruff (0.11.9)

Unused method argument: `client`

(ARG002)